crates/subspace-farmer/src/single_disk_farm/farming.rs

```diff
@@ -261,6 +292,8 @@ where
                 erasure_coding: &erasure_coding,
                 maybe_sector_being_modified,
                 table_generator: &table_generator,
+                audit_event_handler: Some(&handlers.plot_audited),
+                farm_id,
```

Comment:
What do you think about moving metrics to this level instead? We have proving a few lines below, it would be possible to measure that as well and will not require any changes to benchmarking code.